packages/diagrams/frontend/sirius-components-diagrams/src/renderer/layout/useDistributeElements.types.ts

```diff
@@ -25,4 +25,5 @@ export interface UseDistributeElementsValue {
   arrangeInRow: (selectedNodeIds: string[]) => void;
   arrangeInColumn: (selectedNodeIds: string[]) => void;
   arrangeInGrid: (selectedNodeIds: string[]) => void;
+  makeNodeSameSize: (selectedNodeIds: string[], refElementId: string | null) => void;
```

Comment:
Shouldn't it be `makeNodesSameSize` as it involves at least two nodes?

Reply:
done